x-pack/test/alerting_api_integration/security_and_spaces/group3/tests/maintenance_window/find_maintenance_windows.ts

```diff
@@ -90,6 +90,70 @@ export default function findMaintenanceWindowTests({ getService }: FtrProviderCo
               throw new Error(`Scenario untested: ${JSON.stringify(scenario)}`);
           }
         });
+
+        it('should handle find maintenance window request with pagination', async () => {
+          const { body: createdMaintenanceWindow1 } = await supertest
+            .post(`${getUrlPrefix(space.id)}/internal/alerting/rules/maintenance_window`)
+            .set('kbn-xsrf', 'foo')
+            .send(createParams);
+
+          const { body: createdMaintenanceWindow2 } = await supertest
+            .post(`${getUrlPrefix(space.id)}/internal/alerting/rules/maintenance_window`)
+            .set('kbn-xsrf', 'foo')
+            .send(createParams);
+
+          objectRemover.add(
+            space.id,
+            createdMaintenanceWindow1.id,
+            'rules/maintenance_window',
+            'alerting',
+            true
+          );
+          objectRemover.add(
+            space.id,
+            createdMaintenanceWindow2.id,
+            'rules/maintenance_window',
+            'alerting',
+            true
+          );
+
+          const response = await supertestWithoutAuth
+            .get(
+              `${getUrlPrefix(
+                space.id
+              )}/internal/alerting/rules/maintenance_window/_find?page=1&per_page=1`
+            )
+            .set('kbn-xsrf', 'foo')
+            .auth(user.username, user.password)
+            .send({});
+
+          switch (scenario.id) {
+            case 'no_kibana_privileges at space1':
+            case 'space_1_all at space2':
+            case 'space_1_all_with_restricted_fixture at space1':
+            case 'space_1_all_alerts_none_actions at space1':
+              expect(response.statusCode).to.eql(403);
+              expect(response.body).to.eql({
+                error: 'Forbidden',
+                message: 'Forbidden',
+                statusCode: 403,
+              });
+              break;
+            case 'global_read at space1':
+            case 'superuser at space1':
+            case 'space_1_all at space1':
+              expect(response.body.total).to.eql(1);
+              expect(response.statusCode).to.eql(200);
+              expect(response.body.data[0].title).to.eql('test-maintenance-window');
```

Comment:
nit: Could we also test `expect(response.body.data[0].id).to.eql(createdMaintenanceWindow1.id);`. Maybe you can remove the other `body` checks and keep the id and the title as we mostly care about pagination.